src/tunes.js

```diff
@@ -12,8 +12,9 @@ export default class Tunes {
    * @param {object} tune.api - Editor API
    * @param {Function} tune.onChange - tune toggling callback
    */
-  constructor({ api, onChange }) {
+  constructor({ api, actions, onChange }) {
```

Comment:
please describe the new parameter in jsdoc

Reply:
I've updated the code!